crates/iota/src/name_commands.rs

```diff
@@ -393,8 +407,29 @@ impl NameCommand {
                         iota_names_config.package_address, iota_names_config.object_id
                     ),
                     "--assign nft".to_string(),
-                    "--transfer-objects [nft] sender".to_string(),
                 ];
+                if let Some(identity) = set_target_address {
+                    let identity = (identity != "true")
```

Comment:
could we just use `""` instead of `"true"`?

Reply:
no because it's either `"true"` or an address

Reply:
yeah, what I meant was: why not just setting `default_missing_value = ""` in the attribute?

Reply:
I would simply not put a `default_missing_value` at all?

Reply:
Disregard my previous message, it's tristate kind of thing

Reply:
`None`, `Some("true")`, `Some("0x...")`. Was just wondering whether we could go with `None`, `Some("")`, `Some("0x..")`, but if "true" is okay for everyone, we'll go with that then.

Reply:
`None`, `Some(None)`, `Some(Some("0x..."))` :trollface: